scss/styles/navdrawer.scss

```diff
@@ -4,10 +4,16 @@
         color: $white;
     }
 
-    .list-group li {
-        &.home,
-        &.calendar,
-        &.privatefiles {
+    .list-group {
+        // Admin link spacing.
+        [href$='/admin/search.php'] {
+            margin-top: 0.5rem;
+        }
+
+        // Remove home, calendar and private files link using url (classes no longer exist).
+        [href$='/?redirect=0'],
```

Comment:
This is fine since the specificity is high (i.e. it's a list group in the nav drawer)

Reply:
Hopefully at some point we can get some mockups using the new layout so we no longer need to use it.